targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F439xI/TARGET_WIO_BG96/CMakeLists.txt

```diff
@@ -4,8 +4,6 @@
 target_sources(mbed-core
     INTERFACE
         ONBOARD_QUECTEL_BG96.cpp
```

Comment:
As there is not much in this file, we might as well delete this file and move the content to the file including it.